components.md

```diff
@@ -94,11 +94,18 @@ sequenceDiagram
 :::
 
 ## Coordinator
-A component primarily concerned with routing/coordinating the message flow between other Components.
+A component primarily tasked with routing/coordinating the message flow between other Components.
 It represents the intermediate zmq brokers, proxies or somesuch.
```

Comment:
Depending on how we decide to wire the logging and data protocols, _all_ 3 might be in the Coordinator, hence the plural. Also "be" is I think too strong a formulation - it implies an "is-a" relationship in the OOP sense, i.e. inheritance, and I want to avoid that.

Reply:
I used your wording now.